backend/src/helpers/auth.js

```diff
@@ -0,0 +1,7 @@
+const authPreHandler = async (isAuth) => {
```

Comment:
maybe we should propose a structure for the backend and frontend of folders, and intention of folders, so that all of our ideas line up.  for example should this go into helpers, perhaps,